simulator/pages/seir.py

```diff
@@ -296,6 +296,9 @@ def plot_deaths(model_output, scale, start_date, lethality_mean, subnotification
     _, _, _, R, t = model_output
     R /= subnotification_factor
     R *= (lethality_mean/100)
+    _ = R[0]
+    R = np.diff(R, axis=0)
+    R = np.insert(R, 0, _, axis=0)
```

Comment:
there's a `prepend` argument in `np.diff`

https://numpy.org/doc/stable/reference/generated/numpy.diff.html

Reply:
thanks for the advice, much cleaner code